### DISCUSSION TITLE
list all collision domains

### DISCUSSION THREAD

**Discussion by claudiotam** [Kidel/Kathara#310] [Q&A]
Hello!
Is there some command to list all collision domains (and their machines)?

I'm using kathara to create basic labs. After lab creation, I am able to add and remove collision domains with "lconfig".
But after the changes, I can't find a command to list all collision domains. I tried "linfo" and "list" but they don't show collision domains.

**Comment by tcaiazzi** [comment]
Dear @claudiotam, 

You are right, currently, there is no CLI command to display how devices are attached to collision domains.  Probably we should add something similar in the next releases.  

As an example, we can add a column to the `linfo` command specifying the attached collision domains for each device.

Thanks for the suggestion!

Meanwhile, you can understand how devices are connected from the `lab.conf` file. Obviously, dynamic changes on the network (using the `lconfig` command) will not be registered in the file, so you need to remember them for now 😇
